modules/ROOT/pages/constraints/managing-constraints.adoc

```diff
@@ -843,11 +813,17 @@ There are no valid index configuration values for the constraint-backing range i
 === Handling existing constraints when creating a constraint
 
 Creating an already existing constraint will fail.
+This includes the following scenarios:
+
+* Creating a constraint identical to an already existing constraint.
+* Creating a constraint with a different name but on the same schema as an already existing constraint.
+* Creating a constraint with the same name but on a different schema as an already existing constraint.
+
 To avoid such an error, `IF NOT EXISTS` can be added to the `CREATE` command.
-This will ensure that no error is thrown and that no constraint is created if any other constraint with the given name, or another node property uniqueness constraint on the same schema, already exists.
+This will ensure that no error is thrown and that no constraint is created if any other constraint with the given name, or another constraint on the same constraint type and schema, or both, already exists.
```

Comment:
```suggestion
This will ensure that no error is thrown and that no constraint is created if any other constraint with the given name, or another constraint on the same constraint type and schema, or both, already exists.
For property type constraints the property type also needs to be the same.
```
to actually have the information about property types as well (was part of my previous comment as well, but missed)